include/deal.II/grid/grid_reordering.h

```diff
@@ -1,120 +0,0 @@
-// ---------------------------------------------------------------------
-//
-// Copyright (C) 2000 - 2023 by the deal.II authors
-//
-// This file is part of the deal.II library.
-//
-// The deal.II library is free software; you can use it, redistribute
-// it, and/or modify it under the terms of the GNU Lesser General
-// Public License as published by the Free Software Foundation; either
-// version 2.1 of the License, or (at your option) any later version.
-// The full text of the license can be found in the file LICENSE.md at
-// the top level directory of deal.II.
-//
-// ---------------------------------------------------------------------
-
-#ifndef dealii_grid_reordering_h
-#define dealii_grid_reordering_h
-
-
-#include <deal.II/base/config.h>
-
-#include <deal.II/grid/tria.h>
-
-#include <vector>
-
-DEAL_II_NAMESPACE_OPEN
-
-/**
- * A class implementing various grid reordering algorithms. For more information
- * see the
- * @ref reordering "reordering module".
- *
- * @deprecated Use GridTools::invert_all_negative_measure_cells() or
- * GridTools::consistently_order_cells() instead of the functions provided by
- * this class. Usage of the old-style numbering is deprecated.
- */
-template <int dim, int spacedim = dim>
-class GridReordering
```

Comment:
It's unfortunate that we didn't mark this class as deprecated, nor that we poisoned the header file. Any thoughts?

Reply:
I think it is ok to delete this function, given that all functions in the class were deprecated before.

Reply:
Sounds good!